chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/TestDataDTO.java

```diff
@@ -3,35 +3,23 @@
 import java.io.Serializable;
 
 import gov.healthit.chpl.entity.TestDataEntity;
-
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
```

Comment:
Are we getting rid of the DTOs in favor of the domain objects / entities? Or am I rehashing the same conversation again?